Cargo.toml

```diff
@@ -11,21 +11,36 @@ categories = ["authentication", "web-programming::http-server"]
 keywords = ["actix-web", "middleware", "authentication", "jwt", "keycloak"]
 license = "MIT"
 
+[[example]]
+name = "custom_claims"
+path = "examples/custom_claims.rs"
+required-features = ["paperclip_compat"]
+
+[[example]]
+name = "paperclip"
+path = "examples/paperclip.rs"
+required-features = ["paperclip_compat"]
+
+[[example]]
+name = "simple"
+path = "examples/simple.rs"
+required-features = ["paperclip_compat"]
```

Comment:
This is not required because the `paperclip_compat` is not needed for this example